masterlist.yaml

```
@@ -8239,6 +8239,28 @@ plugins:
       - crc: 0x6F9705F8
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'Ayleid Loot EXtension( - OOO Dungeons)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/22795/'
+        name: 'Ayleid Loot EXtended'
+  - name: 'Ayleid Loot EXtension.esp'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Oscuro''s Oblivion Overhaul' ]
+        condition: 'active("Oscuro''s_Oblivion_Overhaul.esm") and not active("Ayleid Loot EXtension - OOO Dungeons.esp")'
+    tag: [ -Invent.Remove ]
+    clean:
+      - crc: 0xB2EF0FCD
+        util: 'TES4Edit v4.0.4b'
+  - name: 'Ayleid Loot EXtension - OOO Dungeons.esp'
+    msg:
+      - <<: *alsoUseX
+        subs: [ 'Ayleid Loot EXtension.esp' ]
+        condition: 'not active("Ayleid Loot EXtension.esp")'
```

Even though you technically can, it doesn't make much sense to use the OOO patch on its own, so a requirement would be fine here.

```suggestion
    req: [ 'Ayleid Loot EXtension.esp' ]
```